clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -168,6 +168,10 @@
                                                         + "Implementing the <code>ConsumerInterceptor</code> interface allows you to intercept (and possibly mutate) records "
                                                         + "received by the consumer. By default, there are no interceptors.";
 
+    /** <code>max.poll.records</code> */
+    public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
+    public static final String MAX_POLL_RECORDS_DOC = "The maximum number of records to return in a single call to poll(). The default (-1) sets no limit.";
```

Comment:
Looks like we're a little inconsistent about this in this class, but yeah, I think it should.